auth/index.test.js

```diff
@@ -50,151 +49,243 @@ describe("TKHQ", () => {
+          "szrFBNGDkhXyVvRoqjjDT6xd7kRhDXHmtQH3NVkPuVVkeiPFjn6UkyjbiTzuxH9wKH4QdEJUaWxZLM1ZLzByUFN1TNjxVh5aoZENCnKYrSEdZBnRWcK"
+        )
+      )
+    ).toEqual(
+      Array.from(
+        TKHQ.uint8arrayFromHexString(
+          "02cb30f1f44d411383cc2a7bb7135d87e0fbf265d0e002b460c9d38d97b14cd0d26114254d213cd77887293644d942a62516a3f174f01ed1ccb57dea1f8ac88664759bb6febcd8b060e7a11d23c614dd66"
+        )
+      )
     );
-  })
+  });
 
   it("contains uint8arrayToHexString", () => {
-    expect(TKHQ.uint8arrayToHexString(new Uint8Array([0x62, 0x75, 0x66, 0x66, 0x65, 0x72]))).toEqual("627566666572");
-  })
-  
+    expect(
+      TKHQ.uint8arrayToHexString(
+        new Uint8Array([0x62, 0x75, 0x66, 0x66, 0x65, 0x72])
+      )
+    ).toEqual("627566666572");
+  });
+
   it("contains uint8arrayFromHexString", () => {
-    expect(TKHQ.uint8arrayFromHexString("627566666572").toString()).toEqual("98,117,102,102,101,114");
-    
+    expect(TKHQ.uint8arrayFromHexString("627566666572").toString()).toEqual(
+      "98,117,102,102,101,114"
+    );
+
     // Error case: bad value
     expect(() => {
       TKHQ.uint8arrayFromHexString({});
-    }).toThrow('cannot create uint8array from invalid hex string');
+    }).toThrow("cannot create uint8array from invalid hex string");
     // Error case: empty string
     expect(() => {
       TKHQ.uint8arrayFromHexString("");
-    }).toThrow('cannot create uint8array from invalid hex string');
+    }).toThrow("cannot create uint8array from invalid hex string");
     // Error case: odd number of characters
     expect(() => {
       TKHQ.uint8arrayFromHexString("123");
-    }).toThrow('cannot create uint8array from invalid hex string');
+    }).toThrow("cannot create uint8array from invalid hex string");
     // Error case: bad characters outside of hex range
     expect(() => {
       TKHQ.uint8arrayFromHexString("oops");
-    }).toThrow('cannot create uint8array from invalid hex string');
+    }).toThrow("cannot create uint8array from invalid hex string");
     // Happy path: if length parameter is included, pad the resulting buffer
     expect(TKHQ.uint8arrayFromHexString("01", 2).toString()).toEqual("0,1");
     // Happy path: if length parameter is omitted, do not pad the resulting buffer
     expect(TKHQ.uint8arrayFromHexString("01").toString()).toEqual("1");
-  })
+  });
 
   it("contains bigIntToHex", () => {
     expect(TKHQ.bigIntToHex(BigInt(1, 1))).toEqual("1");
     expect(TKHQ.bigIntToHex(BigInt(1), 2)).toEqual("01");
     expect(TKHQ.bigIntToHex(BigInt(1), 4)).toEqual("0001");
     expect(TKHQ.bigIntToHex(BigInt(23), 2)).toEqual("17");
     expect(TKHQ.bigIntToHex(BigInt(255), 2)).toEqual("ff");
-    expect(() => { TKHQ.bigIntToHex(BigInt(256), 2) }).toThrow("number cannot fit in a hex string of 2 characters");
-  })
+    expect(() => {
+      TKHQ.bigIntToHex(BigInt(256), 2);
+    }).toThrow("number cannot fit in a hex string of 2 characters");
+  });
+
+  it("contains bigIntToBase64Url", () => {
```

Comment:
the rest of the above is formatting; this is the net-new test suite collection that needs review :)

Reply:
here's a separate PR to deal with just prettifying: https://github.com/tkhq/frames/pull/50